README.md

```diff
@@ -19,6 +19,7 @@ Every argument is optional.
 | `package`            | The package whose API to check for semver (in Package Id Specification format, see https://doc.rust-lang.org/cargo/reference/pkgid-spec.html for reference). If not set, all packages defined in the Cargo.toml file are processed. | |
 | `manifest-path`      | Path to Cargo.toml of crate or workspace to check. If not specified, the action assumes the manifest is under the default [`GITHUB_WORKSPACE`](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) path. | |
 | `verbose`            | Enables verbose output of `cargo-semver-checks`. | `false` |
+| `rust-toolchain`     | Rust toolchain name to use, e.g. `stable`, `nightly` or `1.68.0`. It will be installed if necessary and used regardless of local overrides and the `rust-toolchain.toml` file. However, if the input is set to be an empty string, the action assumes some Rust toolchain is already installed and uses the default one. | `stable` |
```

Comment:
Can you add a test to make sure a local `rust-toolchain.toml` in the project won't override this setting?

I know it's a bit annoying to test, but it'll be a lot better than debugging it later if it turns out to be a problem...

Reply:
Sure, it's very good idea to have a test for everything that is stated in docs (as long as it's possible). I added it!